### PR TITLE
Release 57.0.0

### PR DESCRIPTION
**Pull request by MajorLift**
This is the release candidate for v57.0.0:
- `@metamask/snaps-controllers@9.3.0` (minor)
- `@metamask/snaps-execution-environments@6.6.1` (patch)

### CODE REVIEW

File: packages/snaps-controllers/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Changed
+- Bump `@metamask/base-controller` from `^6.0.0` to `^6.0.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/permission-controller` from `^10.0.0` to `^10.0.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/rpc-errors` from `^6.2.1` to `^6.3.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/snaps-registry` from `^3.1.0` to `^3.2.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/snaps-rpc-methods` from `^9.1.4` to `^10.0.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+- Bump `@metamask/snaps-sdk` from `^6.0.0` to `^6.1.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+- Bump `@metamask/snaps-utils` from `^7.7.0` to `^7.8.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump peer dependency `@metamask/snaps-execution-environments` from `^6.5.0` to `^6.6.1` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
```
Comment (MajorLift) [inline packages/snaps-controllers/CHANGELOG.md:23]:
```suggestion
- Bump `@metamask/snaps-rpc-methods` to `^10.0.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
- Bump `@metamask/snaps-sdk` to `^6.1.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
- Bump `@metamask/snaps-utils` to `^7.8.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
- Bump peer dependency `@metamask/snaps-execution-environments` to `^6.6.1` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
```

File: packages/snaps-controllers/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- **BREAKING:** Add `snap_resolveInterface` RPC method to the `SnapInterfaceController` ([#2509](https://github.com/metamask/snaps/pull/2509))
```
Comment (Mrtenz) [inline packages/snaps-controllers/CHANGELOG.md:11]: I believe this change is only breaking in `snaps-rpc-methods` which was already released. Will check with @GuillaumeRx though.

Comment (Mrtenz) [reply packages/snaps-controllers/CHANGELOG.md:11]: Confirmed that this is not breaking.

Comment (MajorLift) [reply packages/snaps-controllers/CHANGELOG.md]: Thanks for clearing that up! Fixed here: https://github.com/MetaMask/snaps/pull/2589/commits/867aa2a909a5dd1f827a1b6fac190e9851358314

File: packages/snaps-controllers/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Bump `@metamask/permission-controller` from `^10.0.0` to `^10.0.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/rpc-errors` from `^6.2.1` to `^6.3.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/snaps-registry` from `^3.1.0` to `^3.2.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/snaps-rpc-methods` to `^10.0.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+- Bump `@metamask/snaps-sdk` to `^6.1.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+- Bump `@metamask/snaps-utils` to `^7.8.0` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump peer dependency `@metamask/snaps-execution-environments` to `^6.6.1` ([#2589](https://github.com/MetaMask/snaps/pull/2589))
+
+### Fixed
```
Comment (Mrtenz) [inline packages/snaps-controllers/CHANGELOG.md:25]: We merged one PR today.

```suggestion
### Fixed
- Handle initial connections for preinstalled Snaps ([#2591](https://github.com/MetaMask/snaps/pull/2591))
```

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**